### PR TITLE
Implemented back to top button

### PR DESCRIPTION
**Pull request by IvanFengJK**
#349

### CODE REVIEW

File: src/components/Styles.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
+  
+export const Button = styled.div`
+    position: fixed; /*Position is fixed regardless of scroll*/
+    left: 90%; /*Position is fixed at 90% from the left*/
+    bottom: 5px; /*Position is 5px from the bottom*/
+    font-size: 3rem; /*Size of button is at 3rem, rem is a unit relative to the root element*/
+    color: orange; /*Colour is orange*/
+`
```
Comment (amirulraziqi) [inline src/components/Styles.js:9]: Do add a white fill for the up-arrow.
image

File: src/components/ScrollButton.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  // Display button and when click carry out scrollToTop
+  return (
+    <Button>
+     <FaArrowCircleUp onClick={scrollToTop} 
+     style={{display: visible ? 'inline' : 'none'}} />
+    </Button>
+  );
+}
+  
+export default ScrollButton;
```
Comment (amirulraziqi) [inline src/components/ScrollButton.js:41]: @wei2912 LGTM, but requesting your review still

**Review by amirulraziqi — COMMENTED** [review]
@wei2912 Apart from the white fill, LGTM! 

Still, requesting your review for this specific PR.

File: package-lock.json [hunk 3]
```diff
@@ -5,6 +5,7 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "mentorship-page",
```
Comment (amirulraziqi) [inline package-lock.json:8]: @IvanFengJK Revert any changes made to this file before pushing to github. `package-lock.json` should be left untouched.

**Review by amirulraziqi — COMMENTED** [review]
@wei2912 Apart from white fill and `package-lock.json`, LGTM!

Still, requesting for your review here

**Review by amirulraziqi — CHANGES_REQUESTED** [review]
We'd rather the `package-lock.json` file not be changed. Could you uncommit changes to the package-lock.json file before pushing once more?

Other than that, everything else LGTM

**Comment by wei2912** [comment]
Superseded by https://github.com/AdvisorySG/mentorship-page/pull/509.

**Pull request closed without merging**